client/blocks/comments/post-comment-list.jsx

```diff
@@ -210,6 +231,28 @@ class PostCommentList extends React.Component {
 							} )
 						}</span> : null }
 				</div> }
+				<SegmentedControl>
+					<SegmentedControlItem
+						selected={ commentsFilter === 'all' }
+						onClick={ this.handleFilterClick( 'all' ) }>{ translate( 'All', { context: 'comment status'} ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'All', { context: 'post status'} )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/filter-comments-on-posts)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `Filter all plugins`
- `Filter label for plugins list`
- `post status`
- `posting status`
- `Tab title`
- `Type of themes to display`

 Would it make sense to reuse one of the above?